ingestion/src/metadata/test_suite/api/workflow.py

```diff
@@ -286,12 +287,14 @@ def _create_runner_interface(self, entity_fqn: str):
         return DataLakeTestSuiteInterface(
             service_connection_config=service_connection_config,
             ometa_client=self.client,
-            df=ometa_to_dataframe(
-                service_connection_config.configSource,
+            dfs=return_ometa_dataframes(
```

Comment:
Any advantage of having that function `return_ometa_dataframes`? i see it only return `ometa_to_dataframe` function and adds no additional logic.

Reply:
its's for the tests to mock, i had to add a method that would return mocked df, i have modified it to add sampling in it as well

Reply:
it now accomodates sampling as well..it will call the ometa method, run sampling logic and return teh sampled dataframe